bindata/manifests/cni-config/sriov-cni-config.yaml

```diff
@@ -25,6 +25,12 @@ spec:
 {{- end -}}
 {{- if .VlanQoSConfigured -}}
   "vlanQoS":{{.SriovCniVlanQoS}},
+{{- end -}}
+{{- if .MinTxRateConfigured -}}
+  "minTxRate":{{.SriovCniMinTxRate}},
```

Comment:
key of this field should be `min_tx_rate` which equals to the [json name](https://github.com/intel/sriov-cni/blob/master/pkg/types/types.go#L21) in SR-IOV CNI type

Reply:
Done. Thanks.